### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by ilovesoup**
Clarify config and fix default value

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tispark/76)

### CODE REVIEW

**Comment by ilovesoup** [comment]
@zhexuany @birdstorm @Novemser PTAL

File: README.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ Below configurations can be put together with spark-defaults.conf or passed in t
 | spark.tispark.grpc.timeout_in_sec |  10 | GRPC timeout time in seconds |
 | spark.tispark.meta.reload_period_in_sec |  60 | Metastore reload period in seconds |
 | spark.tispark.plan.allow_agg_pushdown |  true | If allow aggregation pushdown (in case of busy TiKV nodes) |
-| spark.tispark.plan.allow_index_double |  false | If allow index double read (which might cause heavy pressure on TiKV) |
+| spark.tispark.plan.allow_index_double_read |  false | If allow index double read (which might cause heavy pressure on TiKV) |
 | spark.tispark.index.scan_batch_size |  2000000 | How many row key in batch for concurrent index scan |
-| spark.tispark.index.scan_concurrency |  5 | How many threads per index scan retrieving row keys |
-| spark.tispark.table.scan_concurrency |  10 | How many threads per table scan |
+| spark.tispark.index.scan_concurrency |  5 | Maximal threads for index scan retrieving row keys (shared among tasks in side each JVM) |
```
Comment (zhexuany) [inline README.md]: inside?

Comment (ilovesoup) [reply README.md]: fixed.

**Comment by zhexuany** [comment]
@birdstorm PTAL

**Pull request merged**